Docs/guidance_for_az_next.md

```diff
@@ -1,25 +1,25 @@
-## The guidance for az next
+## Instructions for installing and trying 'az next'
 
 ### How to install
 
-`az next` has been released as an extension, so you can install `az next` directly by executing `az extension add --source https://clirecommendation.blob.core.windows.net/package/next-0.1.0-py2.py3-none-any.whl --yes`.  *(If you have ever installed an older version of `az next`, please execute `az extension remove -n next` to uninstall the older version before installing the new version for `az next`)*
-
-However, you need to make sure that the version of `azure-cli-core` is 2.19.0+ (which can be viewed with `az version`). 
-If you are using an older version, please use `az upgrade` to upgrade.
+1. You need `azure-cli-core` version 2.19.0+. Run`az version`to check your version. Use `az upgrade` to upgrade.
+2. Install `az next` directly by executing `az extension add --source https://clirecommendation.blob.core.windows.net/package/next-0.1.0-py2.py3-none-any.whl --yes`.  
+  *If you have an older version of `az next`, run `az extension remove -n next` to uninstall the older version before installing the new version of `az next`*
+3. Run `az config set core.collect_telemetry=True` to turn on telemetry. This will help us improve az next recommendations based on your usage data
 
 ### How to play
 
-1. When you make an error in executing command, it will recommend the solution most used by others who has the same error. <br/>
-*Pelase note: This kind of recommendation need to turn on telemetry. If you haven not turned it on yet, please run `az config set core.collect_telemetry=True` and try again. If telemetry is not turned on, only this kind of recommendation will be affected, others can be used normally.*
```

Comment:
`core.collect_telemetry` must be enabled in the recommendation of solution type (other types of recommendation are not required). So I personally think it is necessary to give a hint here~

Reply:
I was trying to make it simple and have everyone do it.. they may not know which scenario they will use.. so if it doesn't hurt, we can have everyone turn it on..

Reply:
Yes, we want everyone to turn on Telemetry, so it's good to have a brief description at the beginning in the installation description.

However, I personally think it's also necessary to tell users the impact of not turning on Telemetry when describing the Solution recommendation
Otherwise, when the user does not turn on Telemetry, if there are some unexpected situations for Solution recommendation, which will make it inconvenient for the user to find the reason